### PR TITLE
Fix: Inaccurate Description for Kubernetes Community Values

### LINKED ISSUE #6188: Inaccurate descriptions for the Kubernetes Community Values
Raised this issue on the contributor-site repo as well, but went through the source for that and realized that it populates pages based on info pulled from this repo - https://github.com/kubernetes/contributor-site/issues/252

The pages describing the Kubernetes Community Values have subtitles/descriptions as "Mentoring Initiatives"

https://github.com/kubernetes/community/blob/master/values.md
image

### PR DESCRIPTION
**Pull request by sreekaransrinath**
The description for the Kubernetes Community Values in values.md was inaccurate (the existing description was relevant to mentoring initiatives).

Changed it to better reflect the content.

Fixes #6188

### CODE REVIEW

File: values.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "Kubernetes Community Values"
 weight: 2
 description: |
-  This page indexes all of the mentoring initiatives for contributing to the Kubernetes project. For end user mentoring initiatives, check out KubeCon and other CNCF events and programs.
+  This page indexes all of the community values that are part of the Kubernetes Community culture. The indexed values have evolved over time, pushing our project and peers toward constant improvement.
```
Comment (mrbobbytables) [inline values.md:5]:
```suggestion
  The Kubernetes Community values are the keystone to the ongoing success of the project. These principles guide every aspect of the Kubernetes project.
```

This suggestion mirrors the values description we used on https://k8s.io/community page 👍

**Comment by sreekaransrinath** [comment]
I've amended the commit message that caused the do-not-merge/invalid-commit-message flag to be thrown up.

**Comment by alisondy** [comment]
Hey there,

Just  had a look through, have you pushed through the fix? I just noticed the last commit you  pushed through had a # in it

Thanks for taking the time to look into this, and submit a fix! Appreciate :)

**Comment by sreekaransrinath** [comment]
> Hey there,
> 
> Just had a look through, have you pushed through the fix? I just noticed the last commit you pushed through had a # in it
> 
> Thanks for taking the time to look into this, and submit a fix! Appreciate :)

Ah my bad, missed the # in the first commit. Sending fix now :)

**Comment by alisondy** [comment]
🎉🎉 well done on your first PR into the kubernetes org, and thanks for taking the time to get this updated. Appreciate ❤️❤️

**Pull request merged**